.eslintrc-typescript.yml

```diff
@@ -1,4 +1,10 @@
-rules:
-  "@typescript-eslint/no-unused-vars": "error"
-  "@typescript-eslint/explicit-module-boundary-types": "off"
+extends: [
+    "plugin:@typescript-eslint/recommended",
+  ]
 
+rules:
+  "@typescript-eslint/ban-ts-comment": off
+  "@typescript-eslint/ban-types": off
+  "@typescript-eslint/no-explicit-any": warn
+  "@typescript-eslint/no-unused-vars": warn
+  explicit-module-boundary-types: off
```

Comment:
Why do we have to change our linting rules for bumping the version?

Reply:
Great question! I think our previous setup didn't include all the recommended rules for linting TypeScript. With those added there were some parts of the code the rules did not really agree with. So those were switched off or replaced by warnings for the time being and we can look into getting rid of them later.

Or keeping them, for example the ban-ts-comment, it's not the best practice to rely on those, but as we are not abusing them, I see no point in strictly banning all of those.

Reply:
all good to me!